cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -470,7 +214,113 @@ def close(self):
+    def check_capacity(required_size):
+        total, avail, _ = _get_capacity()
+        if avail <= required_size:
+            return "Ceph capacity not enough for size: %d, available: %d, total: %d" % (required_size, avail, total)
+        return None
+
+def _get_capacity():
+    o = shell.call('ceph df -f json')
+    df = jsonobject.loads(o)
+
+    if df.stats.total_bytes__ is not None :
+        total = long(df.stats.total_bytes_)
```

Ruff (0.14.2)

`long` may be undefined, or defined from star imports

(F405)